test/python/test_instructions.py

```diff
@@ -337,10 +370,11 @@ def test_sign_attestation(
         test_hwm
     )
 
-    attestation = Attestation(
+    attestation = build_attestation(
         op_level=1,
-        op_round=2
-    ).forge(chain_id=main_chain_id)
+        op_round=2,
```

Comment:
same here?

Reply:
The same applies here and in other cases.